lib/iex/lib/iex/autocomplete.ex

```diff
@@ -225,8 +225,10 @@ defmodule IEx.Autocomplete do
 
     for mod <- match_modules(base, module === Elixir),
         parts = String.split(mod, "."),
-        depth <= length(parts) do
-      %{kind: :module, type: :elixir, name: Enum.at(parts, depth - 1)}
+        depth <= length(parts),
+        name = Enum.at(parts, depth - 1),
+        not String.starts_with?(inspect(String.to_atom("Elixir." <> name)), ":") do
```

Comment:
@arjan `Macro.classify_identifier/1` also accepts strings, to make it work we should have used other condition—`!= :other` instead of `== :alias`. That's what @josevalim did in the next commit: https://github.com/elixir-lang/elixir/commit/e47914cc3983a71a3a0c18fac5a4c2919ff25d95.